packages/driver/src/cy/commands/files.ts

```diff
@@ -109,16 +117,15 @@ export default (Commands, Cypress, cy, state) => {
       return verifyAssertions()
     },
 
-    // TODO: change the type of `any` to `Partial<Cypress.WriteFileOPtions & Cypress.Timeoutable>`
-    writeFile (fileName, contents, encoding, options: any = {}) {
+    writeFile (fileName, contents, encoding, options: Partial<Cypress.WriteFileOptions & Cypress.Timeoutable> = {}) {
```

Comment:
It would be easier to review this if we renamed the parameter instead of changing `options` to `_options`. The majority of the changes would be adding the types then.

Reply:
I changed `options` to `userOptions` and `_options` to `options`. 

I was trying not to change the parameter name, because I thought it would be great to generate type definitions automatically from the source code. 

But I realized that it is almost impossible for us because the first `subject` parameter should be specified on our commands but they don't exist in our API. 

So, it doesn't matter to change those names. 

This PR focuses on fixing the type `TODO`s. 

In the next PR, other un-typed `option`s should be renamed to `userOptions` and types should be specified.